RoslynPluginGenerator/CommandLine/CmdLineResources.resx

```diff
@@ -126,12 +126,18 @@
   <data name="ArgDescription_RecurseDependencies" xml:space="preserve">
     <value>/recurse - search for analyzers in target package and any dependencies</value>
   </data>
+  <data name="ArgDescription_RuleXmlFile" xml:space="preserve">
+    <value>/rules:[path to rules xml file]</value>
+  </data>
   <data name="ArgDescription_SqaleXmlFile" xml:space="preserve">
-    <value>/s:[path to sqale xml file]</value>
+    <value>/sqale:[path to sqale xml file]</value>
```

Comment:
I thought we were removing this sqale support?

Reply:
There's a separate ticket for that.

Reply:
Yep sorry I saw that later in the PR (you put a comment) and forgot to remove some comments.